src/blueteam/Archon.java

```diff
@@ -2,34 +2,40 @@
 
 import battlecode.common.Direction;
 import battlecode.common.GameActionException;
-import battlecode.common.MapLocation;
 import battlecode.common.RobotController;
+import battlecode.common.RobotInfo;
 
 public class Archon extends Robot {
 
+	private final int desiredNumberOfGardeners = 10;
```

Comment:
Java constants naming convention: e.g. DESIRED_GARDENERS_COUNT
 (not important, use whichever convention you prefer)